extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -134,11 +163,14 @@ private function getRevisionMock( array $params ) {
 		$rev->expects( $this->once() )
 			->method( 'getTimestamp' )
 			->willReturn( $params['revision']['timestamp'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "timestamp" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)